From c40e67f1b5cd59a1755d451e00578404fd5d4c61 Mon Sep 17 00:00:00 2001
From: Lars Kellogg-Stedman <lars@redhat.com>
Date: Thu, 28 Aug 2014 15:03:01 -0400
Subject: [PATCH] --vlan is not the same as --vlan_start

the lib/puppet/provider/nova_network/nova_manage.rb provider was using
the --vlan option when it meant --vlan_start.  This caused "network
create" commands with num_networks > 1 to fail with "duplicate vlan"
errors.

Change-Id: Ibdca3b8de72b459eacf8b44b6350b9bc71310c1b
Closes-bug: #1362772
---
 lib/puppet/provider/nova_network/nova_manage.rb | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/puppet/provider/nova_network/nova_manage.rb b/lib/puppet/provider/nova_network/nova_manage.rb
index 6eb2937..6da03a3 100644
--- a/lib/puppet/provider/nova_network/nova_manage.rb
+++ b/lib/puppet/provider/nova_network/nova_manage.rb
@@ -30,7 +30,7 @@ Puppet::Type.type(:nova_network).provide(:nova_manage) do
       :dns2             => '--dns2',
       :gateway          => '--gateway',
       :bridge           => '--bridge',
-      :vlan_start       => '--vlan'
+      :vlan_start       => '--vlan_start'
     }.each do |param, opt|
       if resource[param]
         optional_opts.push(opt).push(resource[param])
-- 
1.9.0

